clogMonitor/src/components/LogEventsFilters.js

```diff
@@ -241,9 +260,18 @@ const LogEventsFilters = ({ dataSetHandler }) => {
         if (startTime === "" || endTime === "") {
             return true;
         }
-        if ((new Date(endTime) < (new Date(startTime)))) {
+        if (new Date(endTime) < new Date(startTime)) {
```

Comment:
This needs to be updated to include DST. It might be easiest to reafactor isRangeError() into an exported function (with signature `isRangeError(startTime, endTime)`) in TimeRange.js so that you can just call it here.